archive/forms.py

```diff
@@ -3,24 +3,40 @@
 from .models import Collection, Document, Picture, PublicFile
 
 
+class MultipleFileInput(forms.ClearableFileInput):
+    allow_multiple_selected = True
+
+
+class MultipleFileField(forms.FileField):
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault("widget", MultipleFileInput())
+        super().__init__(*args, **kwargs)
+
+    def clean(self, data, initial=None):
+        single_file_clean = super().clean
+        if isinstance(data, (list, tuple)):
+            result = [single_file_clean(d, initial) for d in data]
+        else:
+            result = single_file_clean(data, initial)
+        return result
+
+
 class PictureUploadForm(forms.Form):
     album = forms.CharField()
-    images = forms.ImageField(required=False, widget=forms.ClearableFileInput(attrs={'multiple': True}))
+    images = forms.ImageField(required=False, widget=forms.ClearableFileInput())
```

Comment:
Should it use the MultipleFileInput?

Reply:
Sure, but does anyone even use that one lol :D Is that not the one obfuscated somewhere in the archive view

Reply:
Yup this one 
![image](https://user-images.githubusercontent.com/13031380/236906031-f849f81f-a575-4f3f-aa96-ce6e69cd7f6f.png)

Reply:
> Sure, but does anyone even use that one lol :D Is that not the one obfuscated somewhere in the archive view

I think I saw someone use that one at some point